configure.ac

```diff
@@ -602,15 +602,33 @@ AC_ARG_WITH([gdrcopy],
 			    and runtime libraries are installed.])],
 	    [], [])
 
-FI_CHECK_PACKAGE([gdrcopy],
-		 [gdrapi.h],
-		 [gdrapi],
-		 [gdr_open],
-		 [-lgdrapi],
-		 [$with_gdrcopy],
-		 [],
-		 [AC_DEFINE([HAVE_GDRCOPY], [1],[gdrcopy support])],
-		 [], [])
+AS_IF([test x"$with_gdrcopy" == x"yes"],
+	[with_gdrcopy="/usr"],
```

Comment:
Do we need to set this, versus just letting the default include path be used?

Reply:
This is needed to handle the case that gdrcopy is installed in `/usr`, and user used `--with-gdrcopy` (without specifying directory). In this case, FI_CHECKOUT_PACKAGE can find gdrcopy, `with_gdrcopy` would be `yes`, and FI_CHECK_PACKAGE would append `include` to `yes` to make `gdrcopy_CFLAGS`. e.g. `gdrcopy_CFLAGS` became `yes/include`

Reply:
None of the other with_blah checks I see do it this way.  They all just expect with_blah to either be a directory, blank, or no.  I don't think we should make this a special case.

Reply:
I made some adjustment in new revision to make the code more clear.
The issue here is `with_xxx` can be either `yes` or a directory name, and it is passed to FI_CHECK_PACKAGE. However,  FI_CHECK_PACKAGE expect input is a directory.
So in theory with_cuda/with_rocr/with_ze are all impacted by this issue. The reason it has not been a problem for with_cuda is because cuda header/library are not in `/usr`. so `./configure --with-cuda` does not work, one has to use `./configure --with-cuda=/usr/local/cuda`.